### PR TITLE
Update Deployments and promotions section

### PR DESCRIPTION
**Pull request by tasharnvb**

### CODE REVIEW

File: how-to-docs/drone-how-to.md [hunk 1]
```diff
@@ -240,18 +240,18 @@ Push the changes to your remote repository.
 You can deploy the build you just pushed with the following command:
 
 ```bash
-$ drone deploy UKHomeOffice/<your_repo> 16 preprod
+$ drone deploy ukhomeoffice/<your_repo> 16 preprod
```
Comment (gambol99) [inline how-to-docs/drone-how-to.md]: is it not case sensitive?

Comment (tasharnvb) [reply how-to-docs/drone-how-to.md]: It wasn't on old drone and I don't see why that would change for new drone

File: how-to-docs/drone-how-to.md [hunk 2]
```diff
@@ -286,11 +286,11 @@ deploy-to-prod:
...
 
 ```bash
-$ drone deploy UKHomeOffice/<your_repo> 15 preprod
-$ drone deploy UKHomeOffice/<your_repo> 16 prod
+$ drone deploy ukhomeoffice/<your_repo> 15 preprod
+$ drone deploy ukhomeoffice/<your_repo> 16 prod
 ```
 
-[Read more on environment and constraints](http://readme.drone.io/0.5/usage/constraints/#environment)
+Read more on [environments](http://docs.drone.io/environment/).
```
Comment (jits) [inline how-to-docs/drone-how-to.md]: Should probably link to the docs for the specific version of Drone we're using (v0.8, I believe).

Comment (tasharnvb) [reply how-to-docs/drone-how-to.md]: I think they stopped doing docs for specific versions after 0.5.

_docs.drone.io_ don't seem to have versions and _readme.drone.io_ seems to only be for v0.5 - drone/docs#257

Comment (jits) [reply how-to-docs/drone-how-to.md]: Ah yes, I see what you mean; it's worth mentioning that we're currently on v0.8 of Drone (OR pointing them to a page where we show version numbers for our services).

Comment (tasharnvb) [reply how-to-docs/drone-how-to.md]: I think we can mention the version number in the installation section (which has not been updated yet).

Comment (jits) [reply how-to-docs/drone-how-to.md]: Sounds good.

**Pull request merged**